src/ngMessages/messages.js

```diff
@@ -24,45 +24,64 @@ var jqLite = angular.element;
  * `ngMessage` and `ngMessageExp` directives.
  *
  * # Usage
- * The `ngMessages` directive listens on a key/value collection which is set on the ngMessages attribute.
- * Since the {@link ngModel ngModel} directive exposes an `$error` object, this error object can be
- * used with `ngMessages` to display control error messages in an easier way than with just regular angular
- * template directives.
+ * The `ngMessages` directive allows individual keys in a key/value collection to each be associated with an element
+ * (or 'message') that will show or hide based on the truthiness of that key's value in the collection. A common use
+ * case for `ngMessages` is for annotating form inputs with error messages using the `$error` object exposed by the
```

Comment:
"is for annotating form inputs with error message" -> is to display error messages for inputs